src/AzureMcp.csproj

```diff
@@ -79,6 +79,7 @@
 
   <ItemGroup>
     <EmbeddedResource Include="Resources\azure-best-practices.txt" />
+    <EmbeddedResource Include="Resources\azd-best-practices.txt" />
```

Comment:
Can you add a new entry to the root README.md next to the azure best practices tool?

Reply:
https://github.com/Azure/azure-mcp?tab=readme-ov-file#%EF%B8%8F-azure-best-practices